app/assets/v2/js/pages/bounty_details.js

```diff
@@ -769,7 +769,8 @@ var do_actions = function(result) {
     const _entry = {
       enabled: true,
       href: github_url,
-      text: gettext('View On Github'),
+      text: gettext('View On Github') +
```

Comment:
is this part of the spec?

Reply:
Well, the spec has a question mark at the end ^^, so I thought the best experience is to show additional information only if the github issue is closed. Open for feedback 👍